WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -90,6 +98,7 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
                                                                 phAssetImageLoaderProvider: { PHImageManager.default() })
         self.productImageUploader = productImageUploader
         self.aiEligibilityChecker = .init(site: ServiceLocator.stores.sessionManager.defaultSite)
+        self.tooltipUseCase = ProductDescriptionAITooltipUseCase(isDescriptionAIEnabled: aiEligibilityChecker.isFeatureEnabled(.description))
```

Comment:
Nit: it seems redundant if we have to create this use case when description AI is not enabled. Should we use the check to create it only when necessary?

Reply:
Done in 8fa8fe32e661f4d32d26f8ef1f19d90111cd3b09